dogfood/tester/Dockerfile

```diff
@@ -0,0 +1,5 @@
+FROM ubuntu:focal as client
```

Comment:
if it doesn't matter too much, i think we should start using jammy jellyfish (where do they come up with these names), it's the latest LTS